website/server/controllers/api-v3/auth.js

```diff
@@ -532,6 +532,9 @@ api.updateEmail = {
     let validationErrors = req.validationErrors();
     if (validationErrors) throw validationErrors;
 
+    let userSameEmail = await User.findOne({'auth.local.email': req.body.newEmail}).exec();
+    if (userSameEmail) throw new NotAuthorized(res.t('cannotFulfillReq'));
```

Comment:
I noticed that we throw a not authorized error in the registration route when the email is taken.  But I'm not sure in this case should we maintain consistency or throw a bad request error instead because we want to protect privacy (?)